proposals/wac-acp-diff-story.md

```diff
@@ -133,3 +135,67 @@ _:bart-copies-assignment-2
   a acl:AccessControl ;
   acp:apply </myAccessPolicies#bart-copies-my-homework> .
 ```
+
+## Minimal Extension to WAC enabling the same features
+
+Another way of getting the same effect as ACP is to extend WAC with a `:authorizes`
+relation defined as 
+
+```Turtle
+acl:accessTo owl:inverseOf [ 
+        owl:propertyChainAxiom ( acl:accessControl :authorizes )
+      ] .
+```
+
+(We leave the namespace for `:authorizes` open for the moment as it could be in the wac, acp, or another namespace).
+
+This would allow one to place rules in different resources without needing to specify the `wac:accessTo` relation as in ACP. So for example the access control resource associated with Tim Berners-Lee's WebID Profile could contain 
+
+```Turtle
+<> :authorizes _:a1, <personal#Rule1> .
+_:a1 a acl:Authorization;
+   acl:agentClass foaf:Agent ;
+   acl:mode acl:Read .
+```
+
+Note that the `acl:accessTo` relation is missing here. But it can easily be inferred. Any agent - be it a client or the server Guard - following the `Link: <card.acl>;rel=acl` header from the original resource `<card>`, can then follow the `:authorizes` links and so deduce the following 2 statements (see the illustration below)
+
+ 1. for the bnode `_:a1`
+
+```Turtle
+_:a1 acl:accessTo <card> .
+```
+
+Here deduction means quite simply: to create a new graph with the above triple added to it. Once that is done, the same `AclShape` written up above holds.
+
+ 2. for the `<presonal#Rule1>` 
+
+As above after dereferencing the `<personal#Rule1>` the same principle applies. 
+If the document `<personal>` contains the following triples:
+
+```Turtle
+<#Rule1> acl:agent <card#i>;
+    acl:mode acl:Read, acl:Write .
+```
+
+then the client having followed the `Link: <card.acl>;rel=acl` header from `<card>` 
+and followed the `:authorizes` link to `<personal>` would be able to deduce the triple
```

Comment:
If you define `owl:propertyChainAxiom ( acl:accessControl :authorizes )` I think we would need `Link: <card.acl>;rel="http://www.w3.org/ns/auth/acl#accessControl"` rather than `Link: <card.acl>;rel="acl"`

Reply:
Ok, I fixed that, and hope to have added a ShEx for WAC+:authorizes (it may well be wrong)